chrome-launcher/chrome-finder.ts

```diff
@@ -46,25 +52,43 @@ export function darwin() {
     {regex: /^\/Applications\/.*Chrome.app/, weight: 100},
     {regex: /^\/Applications\/.*Chrome Canary.app/, weight: 101},
     {regex: /^\/Volumes\/.*Chrome.app/, weight: -2},
-    {regex: /^\/Volumes\/.*Chrome Canary.app/, weight: -1}
+    {regex: /^\/Volumes\/.*Chrome Canary.app/, weight: -1},
+    {regex: new RegExp(process.env.LIGHTHOUSE_CHROMIUM_PATH), weight: 150},
+    {regex: new RegExp(process.env.CHROME_PATH), weight: 151}
   ];
   // clang-format on
 
   return sort(installations, priorities);
 }
 
+function resolveChromePath() {
+  if (canAccess(process.env.CHROME_PATH)) {
+    return process.env.CHROME_PATH as string;
```

Comment:
is the `as string` necessary here if `canAccess()` doesn't complain in the line above?